node/src/BaseClient.ts

```diff
@@ -4610,14 +4610,20 @@ export class BaseClient {
      * @param key - The key of the stream.
      * @param values - field-value pairs to be added to the entry.
      * @param options - options detailing how to add to the stream.
+     * @param options - (Optional) Additional Parameters:
+     * - (Optional) `StreamAddOptoins`: options detailing how to add to the stream.
+     * - (Optional) `decoder`: see {@link DecoderOption}.
```

Comment:
```suggestion
     * @param options - (Optional) See {@link StreamAddOptions} and {@link DecoderOption}.
```
No need to expand these if you don't use anonymous types